.github/workflows/preview.yml

```diff
@@ -30,7 +30,11 @@ jobs:
         run: |
           mdbook build
 
+      # NOTE: we can only deploy PR previews from the original repository.
+      # This action fails when the PR originates from a forked repository;
+      # see https://github.com/rossjrw/pr-preview-action/issues/3 for details.
       - name: Deploy preview
+        if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
I would have expected github.event.pull_request.head.repo.full_name and github.repository to be wrapped in ${{}}. 
Is this not the case? 
I could not think of a way to check this change.

Reply:
Very good question! I don't think I noted precisely where I found this solution, but [this discussion](https://github.com/orgs/community/discussions/25217#discussioncomment-3246902) demonstrates that it works. A [newer comment](https://github.com/orgs/community/discussions/25217#discussioncomment-3246904) suggests an alternative solution. But yeah, the only way I can think to test this is to try to fork **this branch** and then submit a PR? And I'm not 100% sure that would actually test this feature ...

Reply:
I agree, that definitely seems to be how it works. 

I've tried your suggestion in EamonConway:test-feature/improved-pull-requests.
I am happy with the results and am happy for this to be merged to master. 

We can also close EamonConway:test-feature/improved-pull-requests without merging but still keep a record of the results.

Reply:
Thanks very much Eamon!